server/report_builder/README.md

```diff
@@ -183,3 +183,39 @@ For example, instead of `icon1` an alternative name can be used by using a refer
 ```
 
 The entry `icon1` from the existing report can then be used under the name `my_icon_name`.
+
+## Translating reports
+
+Reports now have the option to allow for translations using the same localisation suite we use for front end translations.
+
+This can be implemented in the report by adding the following translation function in place of your text:
+
+```
+{{t(k="label.name", l=lang, n="reports.json", f="Name")}}
+```
+
+Where the letters are short hand for the following:
+- t for translate
+  The name of the function
+- k for key 
+  This is the locale key as is used in front end translations. 
+- l for language
+  This is an optional override for language. the default lang will use the language of the front end user.
+  To override the language of a particular key, you can substitute it in like:
```

Comment:
I thought an improvement would be adding the LocalisationService provider to the translation function somehow. So it would be an optional field in tera, and get_translation would default to the user language.

Then we could just have 
```
{{t(k="label.name", n="reports.json", f="Name")}}
``` 
throughout the html reports for an even less verbose option

However I couldn't quite figure out how!